Makefile

```diff
@@ -76,6 +76,30 @@ package-dashboard-rpm:
 
 	rm -rf ${PWD}/build
 
+### build rpm for apisix-openresty:
+.PHONY: package-apisix-openresty-centos7-rpm
+package-apisix-openresty-centos7-rpm:
+	chmod +x ./build-apisix-openresty-centos7.sh
```

Comment:
Should avoid chmod every time we build it.

Reply:
Sorry, I don't know how to do this.

Reply:
chmod +x ./build-apisix-openresty-centos7.sh locally and submit it.